hubmap/frontend/templates/globus.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
but one of dataportal/views.py -> view refers to globus.html. So redirects to globus default authentication would fail. So in that case, this seems like we fall back to custom login which logs in to globus and then add setting cookies etc. Do you prefer to have such a custom login that uses globus authentication along with setting cookies?